setup.py

```diff
@@ -15,15 +15,16 @@
         'sonic_cli': ['aliases.ini']
     },
     scripts=[
+        'scripts/aclshow',
         'scripts/boot_part',
         'scripts/coredump-compress',
         'scripts/decode-syseeprom',
         'scripts/fast-reboot',
         'scripts/generate_dump',
+        'scripts/lldpshow',
         'scripts/portstat',
         'scripts/sfputil',
         'scripts/teamshow', 
-        'scripts/aclshow',
```

Comment:
Yes, setuptools allows commas after the last entry. It was news to me when I started on the project, too. I'm not the biggest fan of it, but it makes adding and removing entries easier, so I've stuck with the convention.